clippy_lints/src/utils/conf.rs

```diff
@@ -284,6 +284,10 @@ define_Conf! {
     ///
     /// The list of unicode scripts allowed to be used in the scope.
     (allowed_scripts: Vec<String> = vec!["Latin".to_string()]),
+    /// Lint: NON_SEND_FIELD_IN_SEND_TY.
+    ///
+    /// Whether to apply the raw pointer heuristic in `non_send_field_in_send_ty` lint.
+    (enable_raw_pointer_heuristic: bool = true),
```

Comment:
This documentation will currently only be displayed in the lint description. A more generic name and description usually helps with reusability. 
```suggestion
    /// Whether to apply the raw pointer heuristic to determine if a type is `Send`
    (enable_raw_pointer_heuristic_for_send: bool = true),
```
This will rename the value in the `.toml` file. You don't need to change the field inside the lint implementation.